### PR TITLE
Replaces balaclava sprite

### PR DESCRIPTION
**Pull request by Pro-gamer-window-vista10**
## About The Pull Request

replaces the old balaclava sprite with a new fleece balaclava
## Why It's Good For The Game

better sprites good that look much better (imo)
## Testing Photographs and Procedure

<details>
<summary>Testing</summary>
it is in the clothesmate vendor and appears to be functioning correctly

vendortest

ingame sprites seem to be working

testingbala

</details>

<details>
<summary>Sprites</summary>

balaclavainhandbanditlarge

balaclavacomplexfrontlarge

balaclavacomplexbacklarge

balaclavasidelarge

</details>

## Changelog
:cl:
add: new balaclava sprites
del: old balaclava sprites
add: new balaclava_up sprite
/:cl:

### CODE REVIEW

**Comment by Pro-gamer-window-vista10** [comment]
crap forgot to remove some tags

**Comment by imsynthex** [comment]
> crap why did it add all those tags

because when posting the PR you're meant to only keep a part of the changelog that's relevant to your PR:

```
🆑
add: Added a new type of balaclava
/🆑
```

**Comment by Pro-gamer-window-vista10** [comment]
> 

thanks im a bit a inexperienced

File: code/modules/clothing/masks/boxing.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	w_class = WEIGHT_CLASS_SMALL
 	actions_types = list(/datum/action/item_action/adjust)
 
+
```
Comment (Archanial) [inline code/modules/clothing/masks/boxing.dm]: random newline

File: code/modules/mining/abandoned_crates.dm [hunk 2]
```diff
@@ -239,7 +239,7 @@
 		if(96)
 			new /obj/item/hand_tele(src)
 		if(97)
-			new /obj/item/clothing/mask/balaclava
+			new /obj/item/clothing/mask/bandit_balaclava
```
Comment (Archanial) [inline code/modules/mining/abandoned_crates.dm]: Why are we replacing old balaclava here?

**Review by Archanial — CHANGES_REQUESTED** [review]
Post testing proof please
Also better justification is needed. More variety isn't good enough and those items are *almost* the same.

File: code/modules/mining/abandoned_crates.dm (hunk 2 above)
Comment (Pro-gamer-window-vista10) [reply code/modules/mining/abandoned_crates.dm]: i dont remember to be honest i will revert it

File: code/modules/clothing/masks/boxing.dm [hunk 3]
```diff
@@ -64,3 +64,13 @@
 	flags_inv = HIDEFACE|HIDEHAIR|HIDEFACIALHAIR|HIDESNOUT
 	visor_flags_inv = HIDEFACE|HIDEFACIALHAIR|HIDESNOUT
 	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/clothing/mask/bandit_balaclava
+	name = "bandit balaclava"
+	desc = "Often worn by aspiring criminals and bandits"
```
Comment (itsmeow) [inline code/modules/clothing/masks/boxing.dm]: needs a period at the end

File: code/modules/vending/clothesmate.dm [hunk 4]
```diff
@@ -118,6 +118,7 @@
 		            /obj/item/clothing/under/rank/civilian/bartender/purple = 2)
 	contraband = list(/obj/item/clothing/under/syndicate/tacticool = 1,
 		              /obj/item/clothing/mask/balaclava = 1,
+					   /obj/item/clothing/mask/bandit_balaclava = 2,
```
Comment (itsmeow) [inline code/modules/vending/clothesmate.dm]: this is indented oddly

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by itsmeow** [comment]
You could just submit this as a resprite of the existing balaclava. I believe it is supposed to be one. It could also just be called "fleece balaclava" since they kind of look like different fabrics

**Comment by Pro-gamer-window-vista10** [comment]
> You could just submit this as a resprite of the existing balaclava. I believe the original it is supposed to be a 'typical balaclava'. It could also just be called "fleece balaclava" since they kind of look like different fabrics

alright ive made it into a replacer now with the testing tab updated too

**Comment by imsynthex** [comment]
Please add

```
## Changelog
:cl:
imageadd: resprited the balaclava
/:cl:
```

to the end of your PR

**Comment by Pro-gamer-window-vista10** [comment]
> Please add
> 
> ```
> ## Changelog
> :cl:
> imageadd: resprited the balaclava
> /:cl:
> ```
> 
> to the end of your PR

**Comment by Pro-gamer-window-vista10** [comment]
> Please add
> 
> ```
> ## Changelog
> :cl:
> imageadd: resprited the balaclava
> /:cl:
> ```
> 
> to the end of your PR

i accidentaly forgot to add 
</details> to the end of the testing evidence so it was nested inside there

**Review by itsmeow — APPROVED** [review]
Changes look good. the `balaclava_up` icon_state that was removed is entirely unused, so that's good. I like the new sprite, it looks more like an actual balaclava/ski mask.

**Comment by Pro-gamer-window-vista10** [comment]
what could even be conflicting?

**Comment by itsmeow** [comment]
> what could even be conflicting? it just replaces the sprites which probably havent been touched in years

dmis conflict with anything else in the same file. Another PR updated the mask files, so it conflicts.

You can run the "Fix Icon Conflicts" tool in "tools/dmi"

**Review by PestoVerde322 — COMMENTED** [review]
immagine
Update the balaclava_up and it should be good to go.

**Comment by itsmeow** [comment]
Should be an easy fix, just do a git merge, run the tool, and commit

**Comment by Pro-gamer-window-vista10** [comment]
sorry for the mess in the commit history

**Comment by itsmeow** [comment]
Looks like iconmerge removed a duplicate ninja mask, which one is actually shown ingame?

**Comment by Pro-gamer-window-vista10** [comment]
wdym? doesnt it say so in the "checks" details?

**Pull request closed without merging**